### ISSUE TITLE
Staging Review finding: Incorrect use of `tabindex`

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#96656]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** My Education Benefits
**Product:** 22-5490 Form
**Feature:** (Fry/DEA) Updates

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @briandeconinck (Accessibility)

### Description

On [/direct-deposit](https://staging.va.gov/education/survivor-dependent-education-benefit-22-5490/direct-deposit), the "Direct deposit information" heading has a `tabindex="0"` attribute. This makes the heading a keyboard-focusable element.

As keyboard users tab through a page, there's an expectation that an element that's tabbed to is interactive --- which this heading is not. This can lead to user confusion, believing that something may be broken or misunderstanding the intent of each element.

## Recommended action

I'm guessing this was done so that the heading could receive focus when saving bank information, which is good focus management. The right way to do this is to programmatically add `tabindex="-1"` to the heading when saving bank info, then setting focus to the heading. That puts focus in the right place without making it something that could be tabbed to accidentally.

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** Keyboard
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
